### ISSUE TITLE
allow colorprofile change for "Network overview"

### ISSUE THREAD

**Issue by RaysCars** [pi-hole/web#2505]
Network overview  (network.php) appears in shades of green. As a color-blind person, these shades cannot be distinguished. Provide an option to change the color legend of this page. 

Colorblindness affects 1/8 men and maybe 1/100 of women. Choosing a colorpallete for colorblindess is very helpful. 

Thank you!

**Comment by PromoFaux** [comment]
It's something we can look into - do you have any suggestions for alternative pallettes?

**Comment by rdwebdesign** [comment]
I think the problem we have is the huge number of "side-by-side" items, specially on clients graphic.

It's very hard to create a palette with 16 or more different colors/shades and still be able to differentiate them, but we could try.

**Comment by RaysCars** [comment]
15 level https://jacksonlab.agronomy.wisc.edu/2016/05/23/15-level-colorblind-friendly-palette/
16 level https://lospec.com/palette-list/colorblind-16
10 level (pick any color) https://colorkit.org/color-blindness

**Comment by rdwebdesign** [comment]
@RaysCars 

I'm trying to create new themes using a better contrast ratio and I'm trying to use better colors too.
I opened a PR with 2 new themes. _(Note: I didn't touch the graphics because this is not the intention at the moment)_.

Can you please test them and give us a feed back?


If you are interested in testing the new PR, just run:
```console
pihole checkout web high_contrast
```
Then, open the web interface and select the new themes.

When you finish the test (or case you decide the themes are too bad to use them), you need to revert the changes using:
```console
pihole checkout web master
```

**Comment by yubiuser** [comment]
I think this is solved by https://github.com/pi-hole/AdminLTE/pull/2522

@RaysCars if you want to try it, please run

`pihole checkout web devel`


Then, open the web interface and select the new themes.

When you finish the test (or case you decide the themes are too bad to use them), you need to revert the changes using:

`pihole checkout web master`

**Comment by rdwebdesign** [comment]
@RaysCars

We released the new web interface ([`v5.19`](https://github.com/pi-hole/AdminLTE/releases/tag/v5.19)) with 2 new themes.
I hope you have a better experience with the new themes.
Please, let me know your opinion.
